oxcaml/tests/backend/llvmize/id_fn.output

```diff
@@ -0,0 +1,68 @@
+source_filename = "id_fn.ml"
+target triple = "x86_64-redhat-linux-gnu"
+
+@camlId_fn__data_begin = global i64 0
+define void @camlId_fn__code_begin() { ret void }
+
+; camlId_fn__f_0_1_code [ id_fn.ml:1,21--26 ]
+define i64 @camlId_fn__f_0_1_code(i64 %0)  {
```

Comment:
The substring `_0_1_` is not stable, in the sense that the numbers can change with configure options and unrelated compiler changes. This substring  should be filtered out here and in the references to it.  Same problem for per-instruction comments that mention registers, but it'd be easier to just not print the comments. Piggyback on `-dno-asm-comments` for now to determine if comments should be emitted or not.